libsolidity/interface/CompilerStack.cpp

```diff
@@ -1065,6 +1066,7 @@ size_t CompilerStack::functionEntryPoint(
 	shared_ptr<Compiler> const& compiler = contract(_contractName).compiler;
 	if (!compiler)
 		return 0;
+	solAssert(!m_viaIR, "");
```

Comment:
I see that `gasEstimates()` calls this. Does this mean that calling `solc` with both `--gas` and `--experimental-via-ir` will not cause an ICE?

Reply:
Ah, interesting! At least `--gas --experimental-via-ir` is an unimplemented feature then, since `compiler->functionEntryLabel` does not use the IR (`compiler` is a non-IR component).

Reply:
In that case I think we should have a validation against this option combination in `CommandLineParser`.

Reply:
No, we should implement it.

Reply:
Sure but I'd still add a validation until that's implemented.

Or at least the assert here should be changed to `solUnimplementedAssert`.